integration/setup/setup.go

```diff
@@ -3,9 +3,44 @@
 package setup
 
 import (
+	"log"
+	"os"
 	"testing"
+
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/apprclient/integration/teardown"
 )
 
-func WrapTestMain(m *testing.M) {
-	m.Run()
+func WrapTestMain(f *framework.Host, m *testing.M) {
+	var v int
+	var err error
+
+	err = resources(f)
+	if err != nil {
+		log.Printf("%#v\n", err)
+		v = 1
+	}
+
+	if v == 0 {
+		v = m.Run()
+	}
+
+	err = teardown.Teardown(f)
+	if err != nil {
+		log.Printf("%#v\n", err)
+		v = 1
+	}
+
+	os.Exit(v)
+}
+
+func resources(f *framework.Host) error {
+	err := framework.HelmCmd("registry install --wait quay.io/giantswarm/cnr-server-chart:stable -- -n cnr-server")
```

Comment:
We can actually do this with the helm client now can't we? Or do we have an import cycle issue here?

Reply:
The problem here is that we need to install from quay's registry, and for that we need an appr client. I think we can use helmclient together with apprclient instead of `helm registry install` in other repos different from them both, WDYT?

Reply:
Uncertain right now. Lets iterate here. Lets just make sure that we start using our client libs in the integration tests where possible to get rid of the shell outs.